src/game/Library.fs

```diff
@@ -79,30 +81,16 @@ module game =
 
         let isPushingBox =  tile_Δ = Some box || tile_Δ = Some box_on_goal_square 
 
-        let horizontalMove = Δx <>0
```

Comment:
This became a lot easier

Reply:
No checking up and down, no newline stuff, no caring about what line change happened to etc.